uber/models.py

```diff
@@ -591,7 +591,7 @@ def change_badge(self, attendee, badge_type, badge_num=None):
 
                 # determine the new badge number now that the badges have shifted
                 next = self.next_badge_num(badge_type, old_badge_num)
-                badge_num = min(badge_num or next, next)
+                badge_num = min(int(badge_num) or next, next)
```

Comment:
Silly question though just for my own sanity: you DID re-run the unit tests after making this change, right?

I ask because ``badge_num`` can be passed to this function as potentially ``None`` or ``''`` I think.  Or I might be misremembering.  Either way, the unit tests DO pass after making this change, right?